vars/dockerExecute.groovy

```diff
@@ -37,7 +37,10 @@ import groovy.transform.Field
      */
     'dockerEnvVars',
     /**
-     * Name of the docker image that should be used. If empty, Docker is not used and the command is executed directly on the Jenkins system.
+     * Name of the docker image that should be used.
+     * Configure with empty value to execute the command directly on the Jenkins system (not using a container).
+     * Omit to use the default image.
```

Comment:
Maybe [...]: the default image as defined for the step in `resources/default_pipeline_environment.yml `. Not sure if that is too much information. There is always a tradeoff between brevity and having the details